openlibrary/core/lending.py

```diff
@@ -254,25 +254,45 @@ def get_availability_of_editions(ol_edition_ids):
     return get_availability('openlibrary_edition', ol_edition_ids)
 
 @public
-def add_availability(editions):
+def add_availability(items):
```

Comment:
`items` seems like a weak name vs. `editions`.  Is there a more self-documenting name than `items`?

`publications`, `books`?

Reply:
Agreed; unfortunately it's literally used for anything with an `ocaid`-like field in it :/ Editions was more misleading. It could mainly be edition or the solr record representation of a work as normalized by `get_doc`

Reply:
`books` or `docs` may be more specific as they are almost always going to be solr docs (for or with editions, given that is what the `ocaid` represents). This isn't a blocker. I think `items` is fine.

Reply:
I'd rather opt to leave this as is; I spend a decent amount of time thinking this only accepted editions, when it really accept just about anything. I'd rather future devs have to read the code, because there's no one-word variable name I can think of that would make this clear just from the signature alone.